source/_config.yml

```diff
@@ -582,6 +590,19 @@ documents:
     version: "1"
     nav: /_includes/hiera1.html
 
+  /puppetserver/6.3:
+    doc: puppetserver
+    version: "6.3"
+    nav: /puppet/6.3/_puppet_toc.html
```

Comment:
This is Puppet Server 6.3, which is included in Puppet 6.4. If the version number on line 596 refers to a Puppet version, I'll change it to 6.4.